# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1303693976 -43200
# Node ID 35c09f178dabbeac3e3476f588d6a4d06b15be51
# Parent  78fa2343e4fc2b97b5360d8bfaaac03dff3a7ac4
#11244 reenable extra deprecation warning message issued by python' own warn mechanism in sage/combinat/partition.py.

diff --git a/sage/combinat/partition.py b/sage/combinat/partition.py
--- a/sage/combinat/partition.py
+++ b/sage/combinat/partition.py
@@ -2520,6 +2520,7 @@
         [[2, 1, 1, 1]]
     """
     import warnings
+    warnings.resetwarnings()
     warnings.warn('RestrictedPartitions is deprecated; use Partitions with the parts_in keyword instead.', DeprecationWarning, stacklevel=2)
     from sage.misc.misc import deprecation
     deprecation('RestrictedPartitions is deprecated; use Partitions with the parts_in keyword instead.')
@@ -2541,6 +2542,7 @@
             True
         """
         import warnings
+        warnings.resetwarnings()
         warnings.warn('RestrictedPartitions_nsk is deprecated; use Partitions with the parts_in keyword instead.', DeprecationWarning, stacklevel=2)
         from sage.misc.misc import deprecation
         deprecation('RestrictedPartitions_nsk is deprecated; use Partitions with the parts_in keyword instead.')
